### PR TITLE
Create separate `page-report` for locales

### PR DESCRIPTION
**Pull request by cw118**
Create a `page-report` file for each locale, all in English first (w/ updates to CODEOWNERS) so that the yari PR can be merged sooner. See #8918 and mdn/yari#7277. The idea is that locale teams can translate templates after this and the yari update are merged.

Politely poking @mdn/mdn-community-engagement for a review :)

### CODE REVIEW

File: .github/CODEOWNERS
Comment (nschonni) [inline .github/CODEOWNERS:61]:
```suggestion
/docs/pt-br/  @mdn/yari-content-pt-br
/docs/ru/     @mdn/yari-content-ru
```

**Review by nschonni — COMMENTED** [review]
~~Not really related, just noticed the other change you made~~ missed the difference between the `/docs/` and the existing `/files/`

**Comment by cw118** [comment]
> ~Not really related, just noticed the other change you made~ missed the difference between the `/docs/` and the existing `/files/`

I didn't see the edit of the comment and added `docs` owners for other locales in case translation guides are created for those, I guess we will see what `community-engagement` says. Thank you for the review and credit goes to @yin1999 for first noticing that CODEOWNERS needed an update :)

**Comment by nschonni** [comment]
No problem, we just were updating at the same time. I don't think that the extra entries will cause any issue, I had only realized afterwards that the files didn't exist yet

**Comment by JuanVqz** [comment]
Why is this failing?

**Comment by cw118** [comment]
> Why is this failing?

It's just the system file change workflow, which always fails no matter the changes brought to system files (e.g. `.github` files)

Politely poking @mdn/mdn-community-engagement again for a review :)

**Comment by JuanVqz** [comment]
Thank you for explaining, this is pretty need, I can’t wait to see it merged 🚀 thanks!

**Review by caugner — APPROVED** [review]
LGTM, although everyone will need to be careful not to change the field names (or this could be ensured via a workflow).

**Review by nschonni — APPROVED** [review]
I think the field thing should be fine, since this should still require the `@mdn/core-dev` approval for changes to these files

**Comment by caugner** [comment]
@JuanVqz Good catch! Looks like GitHub doesn't like that all page report templates have the same first line:

```
name: "[via MDN pages only]"
```

The best would probably be to add the locale to it, e.g.:

```
name: "[via MDN pages only // es]"
```

**Comment by caugner** [comment]
I'll draft a PR right away, as this breaks the links.

**Pull request merged**
